app/views/assignment_swap/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <h1><%= current_office.name %> Kitchen Schedule</h1>
 
-<%= form_for(:assignments) do |form|%>
+<%= form_for(:assignments, html: { id: "swapping_form" }) do |form|%>
```

Comment:
I expected to find a `javascript_include_tag(...)` here. How are you including that JS? We should try to load the JS only on this page.